src/Algebra/Graph.hs

```diff
@@ -1010,6 +1010,22 @@ induce p = foldg Empty (\x -> if p x then Vertex x else Empty) (k Overlay) (k Co
     k f x     y     = f x y
 {-# INLINE [1] induce #-}
 
+-- | Construct the /induced subgraph/ of a given graph by removing the 
+-- vertices that are Nothing.
```

Comment:
Let's add single quotes around `Nothing` to make it stand out from the rest of the text.

(Also in other modules.)